src/wallet/rpcdump.cpp

```diff
@@ -119,6 +119,7 @@ UniValue importprivkey(const JSONRPCRequest& request)
         throw JSONRPCError(RPC_WALLET_ERROR, "Cannot import private keys to a wallet with private keys disabled");
     }
 
+    WalletBatch batch(pwallet->GetDBHandle());
```

Comment:
15741: lots of unrelated(?) changes

Reply:
Batching requires the presence of `WalletBatch batch(pwallet->GetDBHandle());` which hasn't been defined and following the spirit of https://github.com/bitcoin/bitcoin/pull/15741/commits/ccb26cf347a6fcc861768bbfbbd652e683ffd5bb, the changes were permeated wherever necessary